frontend/src/domain/SingleGame/SingleGame.tsx

```diff
@@ -0,0 +1,237 @@
+    grid-area: description; 
+    font-size: 16px;
+    font-family: Lao Sangam MN;
+    font-weight: 100;
+
+`
+
+const Categories = styled.div`
+  grid-area: categories;
+
+  display: flex;
+`
+
+const Category = styled.div`
+    font-size: 13px;
+    height: 22px;
+    background-color: ${appTheme.colors.steelGray};
+    text-align: center;
+    padding-top: 6px;
+    padding-left: 20px;
+    padding-right: 20px;
+    margin-right: 15px;
+    margin-top: 10px;
+    z-index:1;
+    border-radius: 12px;
+
+    align-self: center;
+`
+
+const Buttons = styled.div`
+    grid-area: buttons;
+
+`
+
+const Recommended = styled.div`
+    margin-top: 8rem;
+    grid-area: recommended;
+`
+
+const RecommendedCaption = styled.span`
+    border-left: 6px solid ${appTheme.colors.governorBay};
+    padding-left: 16px;
+    font-family: "Rubik";
+    font-size: 20px;
+    font-weight: 500;
+`
+
+const RecommendedItems = styled.div`
+    padding-top: 24px;
+
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap; 
+    justify-content: space-between;
+`
+
+const useStyles = makeStyles((theme) => ({
+    breadcrumbs: {
+      gridArea: "breadcrumbs",
+      color: "white",
+      textDecoration: "none",
+      paddingLeft: theme.spacing(3),
+      fontSize: "1.2rem",
+      fontFamily: "Lao Sangam MN",
+    },
+  
+    element: {
+      color: appTheme.colors.mercury,
+      textDecoration: "none",
+      
+      "&:hover": {
+      color: "white",
+      textDecoration: "underline"
+      }
+    },  
+
+    cover: {
+      gridArea: "cover",
+      width: "650px",
+      alignSelf: "start",
+
+    },
+
+    buyButton:{
+      backgroundColor: appTheme.colors.studio,
+      color: "white",
+      borderRadius: 0,
+      fontFamily: "Lao Sangam MN",
+      fontSize: "15px",
+      textTransform: "none",
+      padding:  theme.spacing(0.7, 2.5),
+      marginRight: theme.spacing(3),
+
+      "&:hover": {
+        backgroundColor: appTheme.colors.fuchsiaBlue,
+      }
+    },
+
+    addButton:{
+      backgroundColor: 'rgba(255, 252, 252, 0.1)',
+      border: `1px solid ${appTheme.colors.ivory}`,
+      color: appTheme.colors.ivory,
+      borderRadius: 0,
+      fontFamily: "Lao Sangam MN",
+      fontSize: "15px",
+      textTransform: "none",
+      padding:  theme.spacing(0.7, 2.5),
+      boxSizing: "border-box",
+    },
+
+  
+  }));
+
+export function SingleGame() {
+    const classes = useStyles();
+    const breadcrumbs = [{to: "/", label: "Home"}, {to: "/insight/adventure", label: "Adventure"}, {to: "/insight/adventure/1", label: "Grand Theft Auto"}];
+    const categories = ['Action', 'Adventure', 'Multiplayer'];
+
+    const item = { title: 'The Witcher', categories: ['Action', 'Adventure'], price: 19.99, cover: witcher} ;
+    const items = [item, item, item, item, item];
+   
+    return (
+      <Home breadcrumbs={breadcrumbs}>
+        <GameContainer>
+        <Header>
+          <Title>
+            GRAND THEFT AUTO
```

Comment:
I think it would be good to decouple those hardcoded values and make them as props of the component in order to easily migrate to server data in the future